### PR TITLE
Add catalog reference to apps in releases

### PR DESCRIPTION
**Pull request by MarcelMue**
## Checklist

- [ ] Consider SIG UX feedback.
- [x] Update changelog in CHANGELOG.md.
- [x] If adding a new type, ensure that it is added to the [CRD unit tests](https://github.com/giantswarm/apiextensions/blob/d78aba91d578d56ef49f58e336035d35edc4e1b0/pkg/crd/crd_test.go#L9).

### CODE REVIEW

**Review by stone-z — APPROVED** [review]

**Comment by MarcelMue** [comment]
Tested as working on ginger

File: config/crd/v1/release.giantswarm.io_releases.yaml [hunk 1]
```diff
@@ -71,6 +71,11 @@ spec:
                 description: Apps describes apps used in this release.
                 items:
                   properties:
+                    catalog:
+                      default: default
```
Comment (rossf7) [inline config/crd/v1/release.giantswarm.io_releases.yaml:75]: Change makes sense to me. Way off topic but I have naming regret. The catalog should be called `preinstalled` 🤦‍♂️ but naming is hard.

**Review by rossf7 — APPROVED** [review]
LGTM

File: config/crd/v1/release.giantswarm.io_releases.yaml (hunk 1 above)
Comment (MarcelMue) [reply config/crd/v1/release.giantswarm.io_releases.yaml:75]: I can relate :D

**Pull request merged**